sessions/sessions.go

```diff
@@ -1583,10 +1585,12 @@ func (sS *SessionS) BiRPCv1AuthorizeEvent(ctx *context.Context,
 		args.APIOpts = make(map[string]interface{})
 	}
 	if v, has := args.APIOpts[utils.OptsAttributeS]; !has {
-		if attrS, err = engine.FilterBoolCfgOpts(ctx, args.Tenant, args.CGREvent.AsDataProvider(), sS.filterS,
-			sS.cgrCfg.CdrsCfg().Opts.Attributes); err != nil {
-			return
-		}
+		/*
+			if attrS, err = engine.FilterBoolCfgOpts(ctx, args.Tenant, args.CGREvent.AsDataProvider(), sS.filterS,
```

Comment:
Are you disabling here the dynamic filters all gogether?

Reply:
I disabled those cases for now because these are taken from sS.cgrCfg.CdrsCfg().Opts(CDRs config), and it should be implementer for sessions config.